src/Routes/api.php

```diff
@@ -23,5 +23,11 @@
                 ->middleware(BadasoAuthenticate::class)
                 ->name('join');
         });
+
+        Route::group(['prefix' => 'courseuser', 'as' => 'courseuser.'], function () {
+            Route::get('/view', HelpersRoute::getController('CourseUserController@view'))
+                ->middleware(BadasoAuthenticate::class)
+                ->name('view');
```

Comment:
nit: isn't it better to put the URL under `/course` since the response contains all courses enrolled by a user. Thus it could be handled by the `CourseController` rather than creating a new controller

Reply:
i will look back into this one next sprint, thanks for the feedback!

Reply:
Actually I agree with Aldi, but isn't this going to eventually bloat `CourseController` in the future?
Even if this should be moved into `CourseController`, I also think it's better to do it in the next sprint

Reply:
I still do believe this is the `CourseController` responsibility` and wouldn't bloat the controller itself